src/views/sandwich-view.tsx

```diff
@@ -30,19 +76,50 @@ class SandwichView extends StatelessComponent<SandwichViewProps> {
   }
 
   componentDidMount() {
+    window.addEventListener('mousemove', this.resizeViews)
+    window.addEventListener('mouseup', this.stopResize)
+    window.addEventListener('mouseleave', this.stopResize)
     window.addEventListener('keydown', this.onWindowKeyPress)
   }
   componentWillUnmount() {
     window.removeEventListener('keydown', this.onWindowKeyPress)
+    window.removeEventListener('mouseleave', this.stopResize)
+    window.removeEventListener('mouseup', this.stopResize)
+    window.removeEventListener('mousemove', this.resizeViews)
   }
 
   render() {
     const {selectedFrame} = this.props
+    let container: JSX.Element | null = null
+    let functionsView: JSX.Element | null = null
+    let resizer: JSX.Element | null = null
     let flamegraphViews: JSX.Element | null = null
 
     if (selectedFrame) {
+      functionsView = (
+        <div
+          className={css(style.tableView)}
+          style={{
+            width: `calc(100% - ${this.state.resizerPos ? this.state.resizerPos + 'px' : '50%'})`,
+          }}
```

Comment:
Because `functionsView`, `resizer`, and `flamegraphViews` are all children of a container with `display: flex`, it shouldn't be necessary to specify all of their sizes changing during the resize. Instead, it should be sufficient only specify one and have the others "flex" to fill the remaining space.

In the code I attached to the top-level comment, I chose to only make `flamegraphViews` resize, and let the `functionsView` "flex" to fill the remaining space.